examples/create_psoxy_connections/variables.tf

```diff
@@ -0,0 +1,44 @@
+variable "resource_name_prefix" {
+  type        = string
+  description = "Prefix to give to names of infra created by this module, where applicable."
+  default     = "worklytics-tenant-api-"
+}
+
+variable "worklytics_tenant_id" {
+  type        = string
+  description = "Numeric ID of your Worklytics tenant's service account (obtain from Worklytics Web App)."
+
+  validation {
+    condition     = var.worklytics_tenant_id == null || can(regex("^\\d{21}$", var.worklytics_tenant_id))
+    error_message = "`worklytics_tenant_id` must be a 21-digit numeric value. (or `null`, for pre-production use case where you don't want external entity to be allowed to assume the role)."
+  }
+}
+
+variable "tenant_api_host" {
+  type        = string
+  description = "Host of the Worklytics Tenant API: the domain by which Cognito will refer users."
+  default     = "intl.worklytics.co"
+}
+
+variable "region" {
+  type        = string
+  description = "The AWS region where Cognito Identity Pool is created."
+  default     = "us-east-1"
+}
+
+variable "user_principal_email" {
+  type        = string
+  description = "The email of the user that has been granted access to the Worklytics Tenant API (configure in Worklytics Web App)."
+}
+
+variable "psoxy_connections" {
```

Comment:
but this should be provided by regular proxy output, if the customer needs to fill this list by hand is a nightmare and not useful at all.
I thought this reused the output of the [worklytics connector](https://github.com/Worklytics/psoxy/blob/rc-v0.5.0/infra/modules/worklytics-psoxy-connection-generic/main.tf) but if that is not the case, it is not very useful